src/main/resources/bundles/commonConfiguration.properties

```diff
@@ -25,6 +25,10 @@
 #file system folder in which marked individual data will be stored (e.g. data files)
 #markedIndividualDirectoryLocation=individuals
 
+animalSingular=whale
+animalPlural=whales
+animalSingularCapitalized=Whale
+animalPluralCapitalized=Whales
 
```

Comment:
This works when you have just one species or species type, but will get awkward if we turn on adoption for something like ACW where there are cats, hyena's, dogs ect. I think that's why for better or worse we just use "animal" in a lot of copy. 

To make this portable you change it to use CommonConfiguration.getIndexedPropertyValues and allow something like
* animalSingular0 = whale
* animalSingular1 = dog

ect ect, but then you would need to parse the returned list with species from the encounter or individual record. Best way to do this would be to make sure the animalSingular and animalPlural lists were in the same order as the genus and species lists. 

I guess we only have single species WB's enabled for adoptions right now so this isn't an urgent fix, but it's worth keeping the limitation in mind.